js/tests.js

```diff
@@ -1,5 +1,43 @@
-QUnit.test("hello test", function(assert) {
-  assert.strictEqual(1 + 1, 2, "One plus one is two");
+QUnit.test("superHero object", function(assert) {
+  var BuffySummers = new Superhero("Buffy", 50, "Angel", true, "I\'m Buffy, the vampire slayer");
+  assert.strictEqual(BuffySummers.loveInterest, "Angel", "gets created and assigns string attribute correctly");
+  assert.strictEqual(BuffySummers.isLoveRequited, true, "assigns boolean attribute correctly");
+  //I don't think I'd have to test all of these; once I know that two data types have been successfully assigned I'm happy
```

Comment:
It's all a matter of risk vs. effort. If you accidentally misspell one of the properties in the constructor, you wouldn't necessarily know without having an assertion for each of them. One way to go about this is to get them all in a single assertion by defining a `properties()` method.

```javascript
assert.deepEqual(buffy.properties(), {
  loveInterest: 'Angel',
  // ...
});
```